scripts/api/data/metadatablocks/citation.tsv

```diff
@@ -238,35 +238,35 @@
 	language	Manx		97	glv	gv
 	language	Macedonian		98	mkd	mac	mk
 	language	Malagasy		99	mlg	mg
-	language	Malay		100	may	msa	ms
+	language	Malay		100	msa	may	ms
 	language	Malayalam		101	mal	ml
```

Comment:
There's a conversation going on where Youn Noh at Yale is proposing these changes:

```
ORIG:
	language	Malayalam		101	mal	ml

NEW:
	language	Malayalam	mal	4016	mal	mal	ml
```

Should we get this change in as well?

Please see https://groups.google.com/g/dataverse-community/c/rGkaHuzA2x4/m/GeyOF1kpAgAJ

Reply:
@pdurbin The change above doesn't make much sense, so no, we certainly don't want it as is. I'm going to add a comment in the google group thread reiterating an earlier suggestion for the author to take a closer look at the opening comment in this PR, since it answers some of the questions asked there.